app/src/store/modules/misc/index.js

```diff
@@ -3,14 +3,16 @@ import actions from './actions.js'
 import getters from './getters.js'
 
 export default {
+  // namespaced: true,
   state () {
     return {
       appHeaderInfo: {
         icon: '',
         pagetype: 'home',
         name: 'MaterialsMine',
         subtitle: 'An open source repository for nanocomposite data (NanoMine), and mechanical metamaterials data (MetaMine)'
-      }
+      },
+      dialogBox: false
```

Comment:
How are you supposed to access the result of the dialog box? Should the result be saved in state?

It may make sense(though not strictly necessary) to make an action `openDialog` that opens the dialog and returns a promise that resolves to the result when the user makes a selection.

Reply:
This is just a scaffold. Since it's inside the state, mutate function is provided by Vuex. But that wasn't implemented as part of this ticket.